### DISCUSSION TITLE
Multiple APs to ad-hoc wireguard clients

### DISCUSSION THREAD

**Discussion by coolstuff99** [RaspAP/raspap-webgui#1538] [Q&A] [answered]
Hi community,

First of all, thanks for this amazing project! worked like a champ during my temporary house move transition few months ago.
I have question related to multiple APs. Would like to have 1 AP for personal us and another AP for guest use. Would like both to have option to enable or disable WG tuneling, but more precisely, ability to select different client to each AP.
This way I can have the guest restricted only to internet access through VPN and personal one not only internet but also access to the local network at home.
Is this something raspap supports or plan to support in the future? I didn't found it in the forum neither as part of the documentation. If it is, apologies in advance.

Thank you,

**Comment by billz** [comment, marked as answer]
Guest networks are often implemented with VLANs or a RADIUS server. This isn't currently supported by RaspAP, but it's a good candidate for a future enhancement. In terms of enabling per-network WG (or OpenVPN) tunneling, this would be an additional piece of work on top of the guest network. Limiting network access to specific clients is an option that already exists in RaspAP (DHCP server > Static leases), however extending this to a per-network configuration is yet another requirement.

There's also UI/UX to consider in order to make all of the above comprehensible to the end user.

Doable, but quite a lot of work.
